### DISCUSSION TITLE
Battery health, throttling and other issues

### DISCUSSION THREAD

**Discussion by jcperil** [Lucchetto/SuperImage#40] [General]
I am wondering what is the battery usage like? Does it throttle while upscaling? I guess I am a little worried. When I tried a similar upscaling software [Upscayl](https://github.com/upscayl/upscayl)  for my laptop, the battery drained quite heavily. And the device started to throttle while upscaling. I uninstalled Upscayl but the battery health deteriorated henceforth.

Can you please address these general issues maybe? Thanks in advance.

**Comment by Lucchetto** [comment]
Hi, most devices have thermal throttling so you don't have worry about your phone blowing up. Unfortunately there is nothing you can do about battery wear
